src/tracks/playabletrack/wavetrack/ui/WaveformView.cpp

```diff
@@ -1143,21 +1132,24 @@ BEGIN_POPUP_MENU(WaveColorMenuTable)
       OnInstrument4ID = hostTable.ReserveId();
    } );
 
-   AppendRadioItem( "Instrument1", OnInstrument1ID,
-      GetWaveColorStr(0), POPUP_MENU_FN( OnWaveColorChange ), fn );
-   AppendRadioItem( "Instrument2", OnInstrument2ID,
-      GetWaveColorStr(1), POPUP_MENU_FN( OnWaveColorChange ), fn );
-   AppendRadioItem( "Instrument3", OnInstrument3ID,
-      GetWaveColorStr(2), POPUP_MENU_FN( OnWaveColorChange ), fn );
-   AppendRadioItem( "Instrument4", OnInstrument4ID,
-      GetWaveColorStr(3), POPUP_MENU_FN( OnWaveColorChange ), fn );
+   for (size_t ii = 0; ii < 3; ++ii) {
```

Comment:
Terminating condition be changed to ii < 4.  It currently is missing the "Instrument 4" menu item.